Subgroup ID vs. encoding (#1468)
moq-wg/moq-transport: draft-ietf-moq-transport.md

0x08 | Group ID field is present | Group ID is the prior Object's Group ID
0x10 | Priority field is present | Priority is the prior Object's Priority
0x20 | Extensions field is present | Extensions field is not present

> 0x40 | Datagram: ignore the two least significant bits | Use the subgroup encoding defined in the two least significant bits

Reviewer: ```suggestion
0x40 | Datagram: ignore the two least significant bits | Decode the Subgroup ID as indicated by the two least significant bits
```
